test/src/089-external_cache_plugin/main

```diff
@@ -0,0 +1,65 @@
+
+cvmfs_test_name="Test CVMFS with external cache plugins"
+cvmfs_test_suites="quick"
+
+CVMFS_TEST_CACHE_PLUGIN_PORT=8081
+CVMFS_CACHE_PLUGIN_POSIX_PATH=/usr/libexec/cvmfs/cache/cvmfs_cache_posix
+CVMFS_CACHE_PLUGIN_RAM_PATH=/usr/libexec/cvmfs/cache/cvmfs_cache_ram
+
+CVMFS_CACHE_PLUGIN_POSIX_PID=""
+CVMFS_CACHE_PLUGIN_RAM_PID=""
+
+cleanup() {
+  if [ "x$CVMFS_CACHE_PLUGIN_POSIX_PID" != "x" ]; then
+    kill $CVMFS_CACHE_PLUGIN_POSIX_PID
+    CVMFS_CACHE_PLUGIN_POSIX_PID=""
+  fi
+  if [ "x$CVMFS_CACHE_PLUGIN_RAM_PID" != "x" ]; then
+    kill $CVMFS_CACHE_PLUGIN_RAM_PID
+    CVMFS_CACHE_PLUGIN_RAM_PID=""
+  fi
+}
+
+cvmfs_run_test() {
+  logfile=$1
+
+  trap cleanup EXIT HUP INT TERM
+
+  mount_opts="CVMFS_CACHE_PRIMARY=externalplugin"
+  mount_opts="$mount_opts CVMFS_CACHE_externalplugin_TYPE=external"
```

Comment:
Isn't there a linebreak missing?

Reply:
it's delimited by space. These opts are later passed to cvmfs_mount()

Reply:
Ah, thanks